pkgs/by-name/cu/cups-idprt/package.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenvNoCC
+, lib
+, fetchurl
+, autoPatchelfHook
+, cups
+, unzip
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "cups-idprt";
+  version = "1.4.2";
+
+  src = fetchurl {
+    url = "https://www.idprt.com/prt_v2/files/down_file/id/131/fid/431.html"; # NOTE: This is NOT an HTML page, but a ZIP file
```

Comment:
We should give it a name that has a zip extension so that it gets unpacked automatically.